src/views/product-integrations-landing/components/faceted-integrations-list/index.tsx

```diff
@@ -1,202 +1,6 @@
-	const matchingOfficial = filteredIntegrations.filter(
-		(i) => i.tier === Tier.OFFICIAL
-	).length
-	const matchingVerified = filteredIntegrations.filter(
-		(i) => i.tier === Tier.PARTNER
-	).length
-	const matchingCommunity = filteredIntegrations.filter(
-		(i) => i.tier === Tier.COMMUNITY
-	).length
-
-	// Pull out the list of all of the components used by our integrations
-	// and sort them alphabetically so they are deterministically ordered.
-	const allComponents = filteredIntegrations.flatMap(
-		(i: Integration) => i.components
-	)
-
-	const mergedComponents = [].concat(allComponents)
-	const componentIDs = mergedComponents.map((c) => c.id)
-	const dedupedComponents = mergedComponents.filter(
-		({ id }, index) => !componentIDs.includes(id, index + 1)
-	)
-	const sortedComponents = dedupedComponents
-		.sort((a, b) => {
-			const textA = a.name.toLowerCase()
-			const textB = b.name.toLowerCase()
-			return textA < textB ? -1 : textA > textB ? 1 : 0
-		})
-		.map((component) => {
-			// Add # of occurances to the component object for facets
-			component.occurances = mergedComponents.filter(
-				(c) => c.slug === component.slug
-			).length
-			return component
-		})
-
-	// We have to manage our component checked state in a singular
-	// state object as there are an unknown number of components.
-	const [componentCheckedArray, setComponentCheckedArray] = useState<boolean[]>(
-		new Array(sortedComponents.length).fill(false)
-	)
-
-	// Now filter our integrations if facets are selected
-	const atLeastOneFacetSelected =
-		officialChecked ||
-		partnerChecked ||
-		communityChecked ||
-		componentCheckedArray.includes(true)
-	if (atLeastOneFacetSelected) {
-		filteredIntegrations = filteredIntegrations.filter(
-			(integration: Integration) => {
-				// Default tierMatch to true if nothing is checked, false otherwise
-				let tierMatch: boolean =
-					!officialChecked && !partnerChecked && !communityChecked
-				if (officialChecked && integration.tier === Tier.OFFICIAL) {
-					tierMatch = true
-				}
-				if (partnerChecked && integration.tier === Tier.PARTNER) {
-					tierMatch = true
-				}
-				if (communityChecked && integration.tier === Tier.COMMUNITY) {
-					tierMatch = true
-				}
-
-				// Loop over each component to see if they match any checked components
-				// If there are no components selected, default this to true
-				let componentMatch = !componentCheckedArray.includes(true)
-				componentCheckedArray.forEach((checked, index) => {
-					if (checked) {
-						const checkedComponent = sortedComponents[index]
-						// Check each integration component
-						integration.components.forEach(
-							(component: IntegrationComponent) => {
-								if (component.slug === checkedComponent.slug) {
-									componentMatch = true
-								}
-							}
-						)
-					}
-				})
-				return tierMatch && componentMatch
-			}
-		)
-	}
-
-	return (
-		<div className={s.facetedIntegrationList}>
```

Comment:
These UI elements have moved to https://github.com/hashicorp/dev-portal/pull/1454/files#diff-bf1326dd3d3e4f79b4f07b95a8b151483551a3352fdae97cfa8fa92b2b9869b5R34 and I stripped their `className`s